nginx/public/node/frontend/public/js/app/manuscript-detail/folio/ChantRecordView.js

```diff
@@ -191,6 +191,12 @@ function get_vowel(texti) {
 	}
 	for (var x = 0; x < strLength(texti); x++) {
 		if (texti[x].toLowerCase() == "a") {
+			// Handle "ae" dipthong
+			if (x < strLength(texti) - 1){
```

Comment:
this is a place where a more descriptive variable name, and perhaps a more descriptive comment at the top of the function, could be helpful. It took me a while to figure out that `texti` is a string representing a single syllable (I think?). Could it be called `syllStr`, `syllChars` or something similar? And the function no longer "returns the number associated with first vowel in an input string" - it returns the number of the vowel the syllable should be sung with.